changelogs/fragments/534-add-support-for-200gb.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - sonic_interfaces and port_group - Add support for 200gb (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/534).
```

Please list each of the module being updated as separate entry.
```suggestion
  - sonic_interfaces - Add support for speed 200GB (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/534).
  - sonic_port_group - Add support for speed 200GB (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/534).  
```

---

Thank you for the review. I have changed the changelog to reflect your feedback.